main/src/ca/uwaterloo/flix/language/phase/Parser.scala

```diff
@@ -442,6 +442,10 @@ class Parser(val source: Source) extends org.parboiled2.Parser {
       SP ~ "\"" ~ zeroOrMore(!("\"" | atomic("${") | atomic("%{")) ~ Chars.CharCode) ~ "\"" ~ SP ~> ParsedAst.Literal.Str
     }
 
+    def Regex: Rule1[ParsedAst.Literal.Regex] = rule {
+      SP ~ "Regex#\"" ~ zeroOrMore(!("\"") ~ Chars.CharCode) ~ "\"" ~ SP ~> ParsedAst.Literal.Regex
```

Comment:
Let us change the syntax to `r"abc"`.

Reply:
Oh. I thought you agreed. What's your proposal?

Reply:
Yes, either one of those is OK for me.

Reply:
(Slight preference for the longer one :) )

Reply:
OK, lets go for the longer one. `regex"foo"`.

Reply:
Updated to `regex"foo"`.